src/Apps/ArtQuiz/Routes/ArtQuizArtworks.tsx

```diff
@@ -220,6 +222,9 @@ export const ArtQuizArtworks: FC<ArtQuizArtworksProps> = ({ me }) => {
     return <ArtQuizResultsLoader onReady={handleReady} />
   }
 
+  // Using split to create a deliberate break in text for better readability
+  const tooltipText = t("artQuizPage.artworksScreen.tooltip").split("\n")
```

Comment:
This is pretty unusual. Do we have no support for multi-line text in the translations? (That seems like it'd be a pretty big oversight)

Reply:
It was actually the tooltip that was causing the issue. You're right about adjusting the tooltip width though. I'll try that instead.